### DISCUSSION TITLE
Issues with the quickstart

### DISCUSSION THREAD

**Discussion by azazel75** [ory/kratos#1899] [Q&A] [answered]
Hello, 

I'm having some issues with the quickstart on the latest  v0.8.0.alpha.2 (BTW, why do you use alphas if you don't tag a final release?) but I don't know if it's my something that does not work on my system (NixOS unstable).

The first is that following the docs at the beginning it says:

> go to 127.0.0.1:4455/dashboard

but the new UI which seems different reports a 404:
> 
>An error occurred
>
>The requested page could not be found (404).

but this is just an issue with the doc lagging behind development

The second is when trying the "sign up" button  on the home page ( http://127.0.0.1:4455/welcome ) on the UI, where the link points to http://127.0.0.1:4455/registration  but then the browser gets redirected to http://kratos:4433/self-service/registration/browser?return_to= but my browser can't resolve the host **kratos** that can be resolved only from the processes running *inside* the containers...

does this happens to you too? Can be quickly fixed?

**Comment by aeneasr** [comment, marked as answer]
Looks like we forgot to update the quickstart properly. Releasing a patch by tomorrow!

**Reply by azazel75** [reply]
Thanks, I'll give it a try!

**Comment by r2evans** [comment]
More things with the quickstart:

- the images have a dark-theme whereas the view on my browser is light; is this just a browser-based preference? I see no ability to lighten/darken it in the actual UI (nor the picture), and my browser (FF-93) *is* on a dark-theme yet the `/welcome` page comes up light-theme
- I cannot find "SecureApp" as a title, just the fact that `:4455` works; is there another endpoint to get to that UI
- (FR) perhaps add a link in the docs to `http://127.0.0.1:4436` for mailslurper
- the quickstart page says *log out and click on "Reset password"*, I see "*Recover Account*" (just a lag in the docs)

**Reply by aeneasr** [reply]
We updated the UI to a new version, which is why it looks and is named different!

It would be awesome if you could help resolve these issues in the docs, would you be open to that? :)

**Reply by r2evans** [reply]
I recognize you only have one keyboard (you can't do everything and resolve everybody's problems), but I am just beginning to learn about the Ory toolset. Having me write documentation is "blind leading the blind" in a sense: while I might be a good reference for pointing out areas of "unstated assumptions", I believe documentation should be written by somebody with more experience in the product.

**Reply by aeneasr** [reply]
> the images have a dark-theme whereas the view on my browser is light; is this just a browser-based preference? I see no ability to lighten/darken it in the actual UI (nor the picture), and my browser (FF-93) is on a dark-theme yet the /welcome page comes up light-theme

So it would just be updating some of the screenshots :) We have updated the application but have not updated the screenshots yet!

> I cannot find "SecureApp" as a title, just the fact that :4455 works; is there another endpoint to get to that UI

You can replace "SecureApp" with "Example App" :) As said above, we updated the app and removed the "Secure App" terminology :)

> (FR) perhaps add a link in the docs to http://127.0.0.1:4436 for mailslurper

You could add that to the quickstart :)

> the quickstart page says log out and click on "Reset password", I see "Recover Account" (just a lag in the docs)

Here too, update the reference you found!

So it is a really good issue to get into the ecosystem and start contributing and be a part of the Ory community! You might even still be eligible for [hacktoberfest](https://hacktoberfest.digitalocean.com) and getting some free swag if you complete some challenges today!
